[CIVIS-9173] Add setup for specifying service in CD Visibility (#21962)
DataDog/documentation: content/en/continuous_delivery/deployments/argocd.md

After this final step is completed, you can start monitoring your Argo CD deployments in Datadog.

## Adding service information to deployment executions

The `dd_service` annotation can be used to map an Argo CD application to the service that it deploys. For example:

```yaml
apiVersion: argoproj.io/v1alpha1
kind: Application
metadata:
  annotations:
    notifications.argoproj.io/subscribe.cd-visibility-trigger.cd-visibility-webhook: ""
    dd_env: <YOUR_ENV>
    dd_service: transaction-service
```

> When this annotation is used, the service name is added to all the deployment executions generated from the application.

Reviewer: ```suggestion
Using this annotation adds the service name to all the deployment executions generated from the application.
```
